### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'Sequelize')

### DISCUSSION THREAD

**Discussion by perfogic** [sequelize/sequelize#15052] [Help & Questions]
Hi everyone I face this error when define one-one relationship hope anyone can give me suggestion to solve it
![image](https://user-images.githubusercontent.com/74671798/192199878-0185dd2b-9a88-4860-817e-189d32d7b759.png)
![image](https://user-images.githubusercontent.com/74671798/192199904-ad5a097c-9928-48dc-998e-015babb21cd9.png)
![image](https://user-images.githubusercontent.com/74671798/192199910-d0c4d95f-36d9-4a92-96ce-2e32aa106eb3.png)

**Comment by pippo111** [comment]
Seems like you didn't import `sequalize`. Line numbers would be helpful and the source of `db` too.

**Reply by perfogic** [reply]
actually I have imported it, and I changed to the other way to define model and it work perfectly

**Reply by perfogic** [reply]
![image](https://user-images.githubusercontent.com/74671798/192566002-9ee5ac2e-1ab9-4cd5-8d6a-e0a7f53be992.png)

**Reply by perfogic** [reply]
changed it to this way will work
![image](https://user-images.githubusercontent.com/74671798/192566563-7ae53e68-995d-4546-b209-92ed45ba51aa.png)

**Comment by ephys** [comment]
You need to call `belongsTo` & other association methods after `init`
The error message could definitely be clearer than what you received

**Reply by perfogic** [reply]
thank you, i will try it again. The doc actually show me like that 
![Screenshot_2022-09-28-08-13-33-641_com.android.chrome.jpg](https://user-images.githubusercontent.com/74671798/192665127-8826b63b-c3ac-4425-8c85-a1534dc465fc.jpg)

**Reply by ephys** [reply]
Oh that's definitely an error, we'll fix that

**Reply by GreenFlag31** [reply]
> You need to call `belongsTo` & other association methods after `init` The error message could definitely be clearer than what you received

Still the same errors thrown by Sequelize, which are very  confusing in term of DX!!
